src/debug-bar.js

```diff
@@ -10,199 +9,24 @@ import { v4 as uuidv4 } from 'uuid';
-    },
-    {
-      key: 'debug-bar-fixture-17',
-      title: 'meeting-room-schedule',
-      file: './fixtures/meeting-room-schedule/meeting-room-schedule.json',
-    },
-    {
-      key: 'debug-bar-fixture-18',
-      title: 'quote',
-      file: './fixtures/quote/quote.json',
-    },
-    {
-      key: 'debug-bar-fixture-19',
-      title: 'poster',
-      file: './fixtures/poster/poster.json',
-    },
-    {
-      key: 'debug-bar-fixture-20',
-      title: 'poster with 2 events',
-      file: './fixtures/poster/poster2.json',
-    },
-    {
-      key: 'debug-bar-fixture-21',
-      title: 'sparkle',
-      file: './fixtures/sparkle/sparkle1.json',
-    },
-    {
-      key: 'debug-bar-fixture-22',
-      title: 'sparkle more posts',
-      file: './fixtures/sparkle/sparkle2.json',
-    },
-    {
-      key: 'debug-bar-fixture-23',
-      title: 'sparkle with video',
-      file: './fixtures/sparkle/sparkle3.json',
-    },
-    {
-      key: 'debug-bar-fixture-24',
-      title: 'rssfeed',
-      file: './fixtures/rss/rss.json',
-    },
-    {
-      key: 'debug-bar-fixture-25',
-      title: 'rssfeed2',
-      file: './fixtures/rss/rss2.json',
-    },
-  ];
-
-  /**
-   * Emit screen data.
-   *
-   * @param {object} screen
-   *   The screen data to emit.
-   */
-  function emitContent(screen) {
-    const event = new CustomEvent('content', {
-      detail: {
-        screen,
-      },
-    });
-
-    document.dispatchEvent(event);
-  }
+  const [screens, setScreens] = useState([]);
 
   /**
    * Load content from fixture.
    *
-   * @param {string} fixture
-   *   The path to the fixture.
+   * @param {string} screenPath
+   *   The path to the screen.
    */
-  function loadContent(fixture) {
-    // Emit empty screen if fixture is null.
-    if (fixture === '') {
-      emitContent({
-        regions: [],
-      });
-      return;
-    }
+  function loadContent(screenPath) {
+    document.dispatchEvent(
+      new CustomEvent('startDataSync', {
+        detail: {
+          screenPath,
+        },
+      })
+    );
 
-    fetch(fixture)
-      .then((response) => response.json())
-      .then((jsonData) => {
-        const screen = {
-          regions: [
-            {
-              id: `region${uuidv4()}`,
-              playlists: [
-                {
-                  id: 'uniquePlaylist1',
-                  slides: [jsonData],
-                },
-              ],
-            },
-          ],
-        };
-
-        emitContent(screen);
-      });
-  }
-
-  /**
-   * Set data synchronization enabled or disabled.
-   *
-   * @param {boolean} enabled
-   *   Boolean. Enabled or disable data synchronization.
-   */
-  function setDataSync(enabled) {
-    if (enabled) {
-      document.dispatchEvent(new Event('startDataSync'));
-    } else {
-      document.dispatchEvent(new Event('stopDataSync'));
-    }
+    setTimeout(() => document.dispatchEvent(new Event('stopDataSync')), 1000);
```

Comment:
Timeout?

Reply:
This is just to stop the sync calls, so it only runs one time when the fixture is loaded.

Reply:
Removed it